src/lcd/ssd1308.hpp

```diff
@@ -36,7 +36,7 @@ const uint8_t DISPLAY_CMD_SET_NORMAL_1308 = 0xA6;
 /**
  * @library i2clcd
  * @sensor ssd1308
- * @comname SSD1308 OLED Display
+ * @comname SSD1308 OLED display
```

Comment:
Hmm, this one is tricky. I would go for resolution instead of SSD1308 but if I recall correctly, the chip supports multiple formats.